pkg/providers/aws/cluster_network_provider.go

```diff
@@ -805,7 +805,7 @@ func (n *NetworkProvider) reconcileStandaloneSecurityGroup(ctx context.Context,
 	}
 	securityGroupTags := ec2TagsToGeneric(standaloneSecGroup.Tags)
 	if !tagsContains(securityGroupTags, tagDisplayName, DefaultRHMISecurityGroupNameTagValue) ||
-		!tagsContains(securityGroupTags, *croOwnerTag.Key, *croOwnerTag.Value) {
+		!tagsContains(securityGroupTags, croOwnerTag.key, croOwnerTag.value) {
```

Comment:
Why did we remove the pointer?

Reply:
Tested this one with a print as suggested and works fine without the pointer. It's related to the change to the return type of getCloudResourceOperatorOwnerTag